scene/3d/skeleton_3d.h

```diff
@@ -91,20 +96,18 @@ class Skeleton3D : public Node3D {
 		Quaternion pose_rotation;
 		Vector3 pose_scale = Vector3(1, 1, 1);
 
-		Transform3D pose_global;
-		Transform3D pose_global_no_override;
+		Transform3D pose;
+		Transform3D pose_override;
+		bool is_override_enabled = false;
 
-		real_t global_pose_override_amount = 0.0;
-		bool global_pose_override_reset = false;
-		Transform3D global_pose_override;
+		// The global_pose should to use for conversion local between global in setter.
+		// So it is just a cache, it shouldn't to use for display.
```

Comment:
// The `global_pose` should be used for conversion to local from global coordinates in the setter.
	// Since `global_pose_cache` is a cache, one shouldn't use `global_pose_cache` for display.